src/hevm/stack.yaml

```
@@ -1,29 +1,23 @@
-resolver: lts-14.14
+resolver: lts-16.11
 packages:
 - .
 extra-deps:
-- config-ini-0.2.4.0
-- brick-0.46
-- data-clist-0.1.2.2
-- free-5.1.2
 - ghci-pretty-0.0.2
 - HSH-2.1.3
 - ipprint-0.6
-- megaparsec-7.0.5
-- multiset-0.3.4.1
-- QuickCheck-2.12.6.1
+- monoidal-containers-0.6.0.1
 - restless-git-0.7
-- rosezipper-0.2
 - s-cargot-0.1.4.0
 - sr-extra-1.46.3.2
+- these-1.0.1 # latest is 1.1.1.1, but monoidal-containers-0.6.0.1 requires <1.1
 - tree-view-0.5
 - text-format-0.3.2
-- text-zipper-0.10.1
 - Unixutils-1.54.1
-- vty-5.25.1
-- word-wrap-0.4.1
+- witherable-0.3.5
 - github: dmjio/semver-range
   commit: d8d9db892ddb6ae267c9bcbc4f6602668433f12a
+- github: leventerkok/sbv # 8.7.5
+  commit: a33239a8bc494282b10426ccf717a8ca635add60
```

Lets use the same revision we are pinning to in our [nix infra](https://github.com/dapphub/dapptools/blob/master/haskell.nix#L18)
```suggestion
  commit: 91637c043d206530bc64d7eac88d2f80e8db0b85
```